src/cis/fam_cis_direct.cpp

```diff
@@ -1013,8 +1013,8 @@ uint64_t Fam_CIS_Direct::get_dataitem_id(uint64_t offset,
     return ((memoryServerId << 32) + offset / MIN_OBJ_SIZE);
 }
 size_t Fam_CIS_Direct::get_addr_size(uint64_t memoryServerId) {
-    CIS_DIRECT_PROFILE_START_OPS()
```

Comment:
Can you add changes related to enabling profiling for prof_fi_sendmsg and others also in this PR.